### ISSUE TITLE
virtio-win-driver-installer setup wizard ended prematurely

### ISSUE THREAD

**Issue by thany** [virtio-win/virtio-win-pkg-scripts#97]
**Describe the bug**
The installer fails to install with this error message: *virtio-win-driver-installer setup wizard ended prematurely*

**To Reproduce**
Install, next, next, next.

**Expected behavior**
Setup succeeds.

**Screenshots**
![image](https://github.com/user-attachments/assets/a58d4838-91ed-4333-a6cc-240ef33063cf)

**Host:**
 - Disto: Unraid 6.12.13
 - Kernel version: 6.1.106 (I think)
 - QEMU version: 7.2.0
 - QEMU command line: no idea
 - libvirt version: 8.7.0
 - libvirt XML file: [qemu.zip](https://github.com/user-attachments/files/17689392/qemu.zip) (taken from Unraid's "xml view")

**Guest:**
 - Windows version: 10.0.19045.5011 
 - virtio-win version or commit hash that was used to build it: 0.1.262-2

**Additional context**
I also got an MSI log for you: [log.zip](https://github.com/user-attachments/files/17689423/log.zip)
Hope it helps.

**Comment by thany** [comment]
Just for kicks, I tried to uninstall the current "virtio-win-guest-tools", which is at 0.1.248, and it fails too:

![image](https://github.com/user-attachments/assets/0b9cf1d9-9b12-41b4-8e00-81d7d3cbfb7e)

This also happens after a successful repair install.

Let's attach the logfile for that as well: [Virtio-win-guest-tools_20241109222523.zip](https://github.com/user-attachments/files/17689438/Virtio-win-guest-tools_20241109222523.zip)

**Comment by JGoutin** [comment]
Same here, all versions most recent than 0.1.229 fail to install.
The installation fails even if I uncheck all components.

**Comment by thany** [comment]
Yeah it feels like we're going to be stuck with whatever version got installed successfully. Just don't expect to ever update it.

Honestly I don't understand how virtio can keep releasing new versions when they can't even bloody install. Like how do they themselves install it? Are these packages even tested *at all*? Like I said at the start, it's a simple matter of "install, next, next, next" and that'll produce the error 100% of the time.
